single-line-diagram-force-layout/src/main/java/com/powsybl/sld/force/layout/ForceSubstationLayout.java

```diff
@@ -77,53 +72,59 @@ public ForceSubstationLayout(SubstationGraph substationGraph,
         this.compactionType = compactionType;
     }
 
-    @Override
-    public void run(LayoutParameters layoutParameters) {
-        // Creating the graph model for the ForceLayout algorithm
-        GraphModel graphModel = new GraphModelImpl();
-        UndirectedGraph undirectedGraph = graphModel.getUndirectedGraph();
-        for (VoltageLevelGraph voltageLevelGraph : getGraph().getNodes()) {
-            NodeImpl n = new NodeImpl(voltageLevelGraph.getVoltageLevelInfos().getId());
-            n.setPosition(random.nextFloat() * 1000, random.nextFloat() * 1000);
-            undirectedGraph.addNode(n);
+    // TODO: move this function in the SubstationGraph class? It implies that the Substation class see the Point and Edge class which is a bit weird?
+    private Graph<Point, Spring> toJgrapht(SubstationGraph graph) {
+        Graph<Point, Spring> pseudograph = new Pseudograph<>(Spring.class);
+
+        // Create nodes
+        for (VoltageLevelGraph voltageLevelGraph : graph.getNodes()) {
+            String id = voltageLevelGraph.getVoltageLevelInfos().getId();
+            Point point = new Point(id, random.nextDouble() * 1000, random.nextDouble() * 1000);
```

Comment:
about the use of random: we need to have the same result from one launch to the other, so we should be able to set the random seed in tests. I'm wondering if it would work as fast (in number of iterations) if we just increase by one the coordinates of each node instead of setting random coordinates?